### PR TITLE
runfix: revert htmlMessage removal from PrimaryModal

### PR DESCRIPTION
**Pull request by tlebon**
## Description
Reverting removal of dangerouslySetInnerHtml as a number of modals will do string interpolation from strings that have manually set links or other custom html in them. 

## Checklist

- [x] PR has been self reviewed by the author;
- [x] Hard-to-understand areas of the code have been commented;
- [x] If it is a core feature, unit tests have been added;

### CODE REVIEW

File: src/script/components/Modals/PrimaryModal/PrimaryModalTypes.ts [hunk 1]
```diff
@@ -79,6 +79,7 @@ export interface ModalContent {
   currentType: string | PrimaryModalType;
   inputPlaceholder: string;
   message: React.ReactNode;
+  messageHtml?: string;
```
Comment (atomrc) [inline src/script/components/Modals/PrimaryModal/PrimaryModalTypes.ts:83]: why do we have both now? what is the diff between `htmlMessage` and `messageHtml`?

Comment (atomrc) [inline src/script/components/Modals/PrimaryModal/PrimaryModalTypes.ts:83]:
```suggestion
  /** @deprecated please use `message` instead */
  messageHtml?: string;
```

Comment (tlebon) [reply src/script/components/Modals/PrimaryModal/PrimaryModalTypes.ts:83]: we had both before as well. depends through which flow the modal is initiated.

Comment (atomrc) [reply src/script/components/Modals/PrimaryModal/PrimaryModalTypes.ts:83]: let's just use `htmlMessage` from now on (and deprecate it)

**Review by atomrc — APPROVED** [review]

**Pull request merged**